### ISSUE TITLE
The program not always recognizes monitor

### ISSUE THREAD

**Issue by Senliast** [xanderfrangos/twinkle-tray#172]
I have an Acer KG241YU monitor with DDC/CI support, and the program recognizes my monitor only after i change its brightness with ControlMyMonitor program. Until that moment the program doesnt recognizes the monitor and says "Make sure that DDC/CI is turned on for the monitors".

**Comment by Armin2208** [comment]
I noticed since the update with some improvement in this area, twinkle tray still randomly doesn't recognizes monitors.
Sometimes after switching from dual screen  to single monitor.
But sometimes just random through the day.

**Comment by Armin2208** [comment]
I correct my comment: seems like right after boot twinkle tray doesn't recognize my monitor. I need to manually close and relaunch it to bring it to work.

**Comment by xanderfrangos** [comment]
Yeah, this is a known issue right now. I'm have a lot of trouble reproducing it, so I have to rely on others for testing.

Could you guys let me know if this version helps?
https://1drv.ms/u/s!AiWKFpjcSKRQhsdeTysDWLhkJRKSDA?e=d1lfj6

**Comment by Armin2208** [comment]
> Yeah, this is a known issue right now. I'm have a lot of trouble reproducing it, so I have to rely on others for testing.
> 
> Could you guys let me know if this version helps?
> https://1drv.ms/u/s!AiWKFpjcSKRQhsdeTysDWLhkJRKSDA?e=d1lfj6

Hi! It's hard so compare them, but this version clearly helps! 😄 
Right after system start this beta version instantly recognizes my monitors. The latest stable not. 
After controlling the brightness with the new version, the old one also recognizes the Monitor lol.

But a new Bug occurs. If I set Win + P to expand, the beta version always shows the second monitor, even if this one is turned off. The old stable behaves correct and only shows the second monitor while it's on. Probably it has something todo with the bug fix, maybe it's needed.

Great work still.

**Comment by xanderfrangos** [comment]
I've released [v1.13.6](https://github.com/xanderfrangos/twinkle-tray/releases/tag/v1.13.6) on GitHub. It should address the issues covered in this Issue (including the Win + P bug). Let me know if it helps. If there aren't any new bugs reported in the next few days, I'll roll it out to the Windows Store as well.

**Comment by horst789github** [comment]
in the new v1.13.7 same problem, sometime the monitor is not recognize in twinkletray settings, i dont no why and how to reproduce

**Comment by Armin2208** [comment]
> in the new v1.13.7 same problem, sometime the monitor is not recognize in twinkletray settings, i dont no why and how to reproduce

was it fixed in 1.13.6 for you?

**Comment by horst789github** [comment]
> 
> 
> > in the new v1.13.7 same problem, sometime the monitor is not recognize in twinkletray settings, i dont no why and how to reproduce
> 
> was it fixed in 1.13.6 for you?

i dont try this version, but i have this problem sometimes in all versions that i have tried...
